<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txt_com">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_com1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Puerto COM:</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_estado">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>ESTADO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_accion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CONECTAR</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lista_datos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="txt_estado_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>DATOS:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_control">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PRENDER</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lista_datos2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lista_datos3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>280</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="txt_estado_4">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>490</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
background-image: url(:/Logos/Castor Ingeniero.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Archivos/Recursos.qrc">:/Logos/Castor Ingeniero.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="txt_estado_5">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
background-image: url(:/Logos/Castor Ingeniero.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Archivos/Recursos.qrc">:/Logos/Facultad.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="txt_estado_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
background-image: url(:/Logos/Castor Ingeniero.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Archivos/Recursos.qrc">:/Logos/UAT.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_control2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>410</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PRENDER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_control3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>410</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PRENDER</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../Archivos/Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
